mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/zBoldTagHelperCopy2.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace AuthoringTagHelpers.TagHelpers2
+{
+[TargetElement("bold", Attributes = "bold")]
```

Comment:
Let's just update this now and add the dedent. That's what we do for all the other articles.